### DISCUSSION TITLE
[Chrome Extension] Refined GitHub：神了

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#142] [1️⃣ 新鲜事]
# Refined GitHub

- Chrome安装地址：https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf
- GitHub仓库地址：https://github.com/refined-github/refined-github

感谢@wxharry觅得此插件！

# 目前有214个features！

超级多的功能，每个功能点都有独立开关。

<img width="715" alt="image" src="https://user-images.githubusercontent.com/32434520/181270555-def56438-cbcd-40c0-94ae-d196d2e253ab.png">

# 这里贴几个功能

1. 快捷at回复 & 显示profile name
![image](https://user-images.githubusercontent.com/32434520/181272614-e1c88dfc-e064-462c-9d0f-554aa6687a1d.png)

2. 快速插入表格
![image](https://user-images.githubusercontent.com/32434520/181271044-eb728832-9c2d-4bcd-87c4-409c19b5ee1c.png)

3. 进入issues列表默认按最近更新排序
![image](https://user-images.githubusercontent.com/32434520/181271273-06a9045f-bef4-4dc5-bdea-4f1ac2057e48.png)

4. 快速checkout到离comment最近的commit
![image](https://user-images.githubusercontent.com/32434520/181271541-74cd5b7f-3c68-4799-b6be-fbbc1c7fdf93.png)

5. 直接看reactions是哪些users点的
![image](https://user-images.githubusercontent.com/32434520/181271941-5a1f707e-b507-4489-af18-8b3f2f3839cb.png)

# 注意事项！

此插件和Hypercrx有冲突，请前往插件option页面关闭“clean-repo-tabs”功能，**否则perceptor tab不会显示出来**。

<img width="324" alt="image" src="https://user-images.githubusercontent.com/32434520/181273603-1c02541d-8c4c-4298-95a3-a4bee0d9e5cc.png">

<img width="566" alt="image" src="https://user-images.githubusercontent.com/32434520/181273758-fe49e883-d407-4592-a991-6261e011ea8e.png">

**Comment by wxharry** [comment]
@zhicheng-ning <- he's the one who find this amazing tool first🤣

**Reply by tyn1998** [reply]
@zhicheng-ning 志成666

**Reply by zhicheng-ning** [reply]
🥲 今天把 chrome 里 GitHub 相关的插件体验了一下
